src/mame/heathkit/tlb.cpp

```diff
@@ -305,38 +305,57 @@ void heath_tlb_device::right_shift_w(int state)
 
 MC6845_UPDATE_ROW(heath_tlb_device::crtc_update_row)
 {
-	if (!de)
-	{
-		return;
-	}
-
 	rgb_t const *const palette = m_palette->palette()->entry_list_raw();
 	uint32_t *p = &bitmap.pix(y);
 
-	for (uint16_t x = 0; x < x_count; x++)
-	{
-		uint8_t inv = (x == cursor_x) ? 0xff : 0;
-
-		uint8_t chr = m_p_videoram[(ma + x) & 0x7ff];
+	if (de) {
 
-		if (chr & 0x80)
+		for (uint16_t x = 0; x < x_count; x++)
 		{
```

Comment:
Please be consistent with your formatting.  You’ve got mixed K&R and Allman here.